sunkit_image/tests/test_enhance.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pytest
+
+import sunpy.map
+import sunpy.data.sample
+from sunpy.tests.helpers import figure_test
+
+import sunkit_image.enhance as enhance
+
+
+@pytest.fixture
+@pytest.mark.remote_data
+def smap():
+    return sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+
+@figure_test
+@pytest.mark.remote_data
+def test_mgn(smap):
+
+    out = enhance.mgn(smap.data)
+    out = sunpy.map.Map(out, smap.meta)
+
+    fig = plt.figure()
+    out.plot()
+
+
+@pytest.fixture
+def map_test():
+
+    A = np.array([[1., 1., 1., 1.],
+                 [1., 5., 5., 1.],
+                 [1., 5., 5., 1.],
+                 [1., 1., 1., 1.]])
+    return A
+
+
+def test_multiscale_gaussian(map_test):
+
+    # Assuming the algorithm works fine then the below two should be equal.
```

Comment:
Why would these two be equal?

Reply:
Since both the kernel sigmas are equal in the second case and for all the images after convolving with kernels, weighted mean is calculated so the final output of first and second will be same.

Reply:
But if we don't plan to verify this manually, we can expand the range of tests.

Reply:
What do you mean by range?

Reply:
I think one test for such a complex function feels too little.